src/main/java/io/micronaut/build/aot/AbstractMicronautAotCliMojo.java

```diff
@@ -41,35 +43,43 @@
 import java.io.File;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static io.micronaut.build.aot.Constants.*;
 import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
 import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
 
 /**
  * Base class for Micronaut AOT mojos.
  */
 public abstract class AbstractMicronautAotCliMojo extends AbstractMicronautAotMojo {
-    private final static String[] AOT_MODULES = new String[]{
+
+    public static final String EXEC_MAVEN_PLUGIN_GROUP = "org.codehaus.mojo";
+    public static final String EXEC_MAVEN_PLUGIN_ARTIFACT = "exec-maven-plugin";
+    public static final String EXEC_MAVEN_PLUGIN_VERSION_PROPERTY = "exec-maven-plugin.version";
+    public static final String DEFAULT_EXEC_MAVEN_PLUGIN_VERSION = "3.0.0";
+
+    private static final String DEFAULT_PACKAGE = "com.example";
+
+    private static final String[] AOT_MODULES = new String[]{
             "api",
             "core",
-            "cli",
             "std-optimizers"
     };
-    public static final String EXEC_MAVEN_PLUGIN_GROUP = "org.codehaus.mojo";
-    public static final String EXEC_MAVEN_PLUGIN_ARTIFACT = "exec-maven-plugin";
-    public static final String EXEC_MAVEN_PLUGIN_VERSION = "3.0.0";
 
     private final ExecutorService executorService;
 
     /**
      * Package name to use for generated sources.
      */
-    @Parameter(property = "micronaut.aot.packageName", required = true)
+    @Parameter(property = "micronaut.aot.packageName", required = true, defaultValue = DEFAULT_PACKAGE)
```

Comment:
Are you sure we want to provide a default value for something which is required?

Reply:
It is not really required in Maven terms, i.e., is not required if the plugin is not enabled. That's why I added a default value to cover when the plugin is not enabled, along with a warning if the parameter has been unset.

Alternatively, I can make it optional, and check manually for it if the plugin is enabled. Now that I think about it, this makes more sense.

Reply:
Done